physics/src/modules/IceOceanHeatFluxModule/BasicIceOceanHeatFlux.cpp

```diff
@@ -27,6 +27,8 @@ void BasicIceOceanHeatFlux::updateElement(size_t i, const TimestepTime& tst)
     // Use the timestep length as the relaxation time scale
     if (cice[i] > 0.) {
         qio[i] = doOne(tf[i], sst[i], mlBulkCp[i], tst.step.seconds());
+    } else {
+        qio[i] = 0.;
```

Comment:
This feels like it should be in its own bug fix PR.

Reply:
Now in #789